{{cookiecutter.project_slug}}/{{cookiecutter.package_slug}}/__main__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Entrypoint module, in case you use `python -m{{cookiecutter.project_name}}`.
+Why does this file exist, and why __main__? For more info, read:
+- https://www.python.org/dev/peps/pep-0338/
+- https://docs.python.org/2/using/cmdline.html#cmdoption-m
+- https://docs.python.org/3/using/cmdline.html#cmdoption-m
+"""
+{%- if cookiecutter.command_line_interface == 'Click' or cookiecutter.command_line_inteface == 'Argparse' %}
+import sys
+
+{% endif %}
+from {{cookiecutter.package_slug}}.cli import main
```

Comment:
Here should it be package_slug or project_name like it is in the __init__ file?

Reply:
just some explanation about package_slug and project_name and project_slug 
https://github.com/osl-incubator/cookiecutter-python/blob/main/cookiecutter.json#L4-L6

slugs in general would work like a codename, a word or words without any empty space in lowercase.

* so project_name would be the name to be displayed in the documentation like "Open Science Labs Cookiecutter" (empty space allowed)
* the project_slug would be the project code name, like the one used by the repository name ... or the package name (for example a python package), which you could install it from pypi or conda, for example: "opensciencelabs-cookiecutter" (dash "-" allowed)
* package_slug is the codename used for the package to be imported, for example: osl_cookiecutter (dash "-" not allowed)

does this explanation answer your question?